applications/asset_tracker_v2/README.rst

```diff
@@ -245,18 +267,33 @@ Check and configure the following configuration options for the application:
 
    This application configuration sets a custom client ID for the respective cloud. For setting a custom client ID, you need to set :option:`CONFIG_CLOUD_CLIENT_ID_USE_CUSTOM` to ``y``.
 
-Additional configuration
-========================
 
-To get the application to communicate with a specified cloud service, configure the Kconfig options specific to each *cloud library*.
-Every cloud service supported in |NCS| has a corresponding *cloud library* that needs to be selected and properly configured.
+.. _mandatory_config:
+
+Mandatory library configuration
+===============================
+
+You can set the mandatory library-specific Kconfig options in the :file:`prj.conf` file of the application.
 
-You must check and configure the following :ref:`lib_aws_iot` library options that are used by the application:
+Configurations for AWS IoT library
+----------------------------------
 
 * :option:`CONFIG_AWS_IOT_BROKER_HOST_NAME`
 * :option:`CONFIG_AWS_IOT_SEC_TAG`
 
-Additionally, you can add the following optional configurations to configure the heap or to provide additional information such as APN to the modem for registering with an LTE network:
+
+Configurations for Azure Iot Hub library
+----------------------------------------
+
+* :option:`CONFIG_AZURE_IOT_HUB_DPS_HOSTNAME`
```

Comment:
I don't think this should be configured by users. The global device provisioning endpoint is the same for all: https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#device-provisioning-endpoint

Reply:
Ah, right :+1: